### DISCUSSION TITLE
Fatal error: Uncaught Error: Class 'Phalcon\Security\JWT\Signer\Hmac' not found

### DISCUSSION THREAD

**Discussion by bilal-karahan** [phalcon/cphalcon#15525] [Q&A]
Hi everyone,

Im learning phalcon but get some error JWT ,the code below ,getting from phalcon 4.0 docs.

`use Phalcon\Security\JWT\Builder;
Use Phalcon\Security\JWT\Signer\Hmac;
use Phalcon\Security\JWT\Token\Parser;
use Phalcon\Security\JWT\Validator;

$signer  = new Hmac();

        $builder = new Builder($signer);

        $now        = new DateTimeImmutable();
        $issued     = $now->getTimestamp();
        $notBefore  = $now->modify('-1 minute')->getTimestamp();
        $expires    = $now->modify('+1 day')->getTimestamp();
        $passphrase = 'QcMpZ&b&mo3TPsPk668J6QH8JA$&U&m2';

        $builder
            ->setAudience('https://target.phalcon.io')  // aud
            ->setContentType('application/json')        // cty - header
            ->setExpirationTime($expires)               // exp
            ->setId('abcd123456789')                    // JTI id
            ->setIssuedAt($issued)                      // iat
            ->setIssuer('https://phalcon.io')           // iss
            ->setNotBefore($notBefore)                  // nbf
            ->setSubject('my subject for this claim')   // sub
            ->setPassphrase($passphrase)                // password
        ;

        $tokenObject = $builder->getToken();

        return $tokenObject->getToken();
`

why dont fing this class ?

**Comment by quasipickle** [comment]
Is other Phalcon code working?  This looks right but wouldn't be defined if Phalcon isn't installed correctly.

**Comment by niden** [comment]
The class is there for sure:

https://github.com/phalcon/cphalcon/blob/master/phalcon/Security/JWT/Signer/Hmac.zep

Otherwise our tests would not pass. I am curious, in your example is the `Use` (capital) a typo or you have that in the code? Maybe that is the cause of the issue?
